template/ios/Podfile

```diff
@@ -25,10 +25,5 @@ target 'HelloWorld' do
 
   post_install do |installer|
     react_native_post_install(installer)
-
-    # Enables Flipper.
-    #
-    # Disable the next line if you are not using Flipper.
-    flipper_post_install(installer)  
```

Comment:
FWIW, I could detect when `flipper_post_install` is placed in a user `Podfile` and print a warning that this is deprecated and one should place `react_native_post_install` instead.

Is it worth it?

Reply:
(just thinking about potential users that might forget to update the `Podfile`)

Reply:
Sure, it might make some users happy to do some Podfile 🧹

Reply:
The technique would be this:
1. Make `flipper_post_install` accept a second optional argument
2. Pass second argument  from within `react_native_pods.rb`, e.g. `flipper_post_install(installer, true)`
3. When second argument is not defined, print a warning that users should replace it with `react_native_post_install` (second argument will not be present unless it is being run by "us")

Not sure if hacky/worth it, but figured I'll just leave it here.

Reply:
I think we can do this in a follow-up PR, if at all.